modules/service.sh

```diff
@@ -29,6 +29,17 @@ service_disable() {
    "${service}_disable" "$@"
 }
 
+service_update() {
+    local service="$1"
+    local token="$2"
+    local bypass_prompt="$3"
+
+    _service_check_user
+    _service_check_support "$service"
+    "${service}_validate_token" "$token" || error_exit invalid_token
+    "${service}_update" "$token" "$bypass_prompt"
```

Comment:
as said above, this wouldn't exist for `esm` or `livepatch`, causing an error

Reply:
As indicated above, I removed it from that file.